### PR TITLE
OpenSCAP cloudapi tailoring

### PR DESCRIPTION
**Pull request by kingsleyzissou**
This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

File: test/cases/oscap.sh [hunk 1]
```diff
@@ -480,7 +486,7 @@ fi
 
 # one grub rule fails (expected)
 # check if any other rules have failed
-if [[ ${SEVERITY} -gt 1 ]]; then
+if [[ ${SEVERITY} -gt 0 ]]; then
   redprint "❌ Failed"
   echo "More than one oscap rule with high severity failed"
```
Comment (bcl) [inline test/cases/oscap.sh:491]: Wording needs to be updated as well.

**Comment by kingsleyzissou** [comment]
Not sure how I managed to close this. I will have to create a new PR

**Comment by kingsleyzissou** [comment]
Somehow I managed to close this PR when I rebased it. :shrug: re-opening

**Comment by kingsleyzissou** [comment]
Somehow I managed to close this PR when I rebased it. 🤷 re-opening

**Comment by henrywang** [comment]
Test `edge-commit-cs9` failure is cause by bug https://bugzilla.redhat.com/show_bug.cgi?id=2234390. I have a workaround PR #3663.

File: test/cases/oscap.sh (hunk 1 above)
Comment (croissanne) [reply test/cases/oscap.sh:492]: I guess this is still an issue?

**Review by croissanne — COMMENTED** [review]
lgtm, @bcl's comment still stands though.

File: test/cases/oscap.sh (hunk 1 above)
Comment (kingsleyzissou) [reply test/cases/oscap.sh:492]: Ah flip, I must have reverted the changes when I lost the commits

Comment (kingsleyzissou) [reply test/cases/oscap.sh:492]: Ah no the diff shows it's been removed. But it's not showing on the review comment weirdly enough.

**Review by thozza — COMMENTED** [review]
Except for my and other comments the PR looks good.

**Comment by kingsleyzissou** [comment]
So I've addressed Brian's comment... but the diff in the comment hasn't updated (and won't update).
But if you look at the commit:

image

File: test/cases/oscap.sh (hunk 1 above)
Comment (kingsleyzissou) [reply test/cases/oscap.sh:492]: Ah flip haha I'm blind. I was too focussed on the diff

**Review by thozza — APPROVED** [review]
LGTM, nice work :slightly_smiling_face:

**Comment by croissanne** [comment]
> So I've addressed Brian's comment... but the diff in the comment hasn't updated (and won't update). But if you look at the commit:
> 
> image

oh I thought it had to do with the comment "More than one oscap rule....", even though it's `-gt 0` now.

But yea not a big deal.

**Comment by kingsleyzissou** [comment]
> > So I've addressed Brian's comment... but the diff in the comment hasn't updated (and won't update). But if you look at the commit:
> > image
> 
> oh I thought it had to do with the comment "More than one oscap rule....", even though it's `-gt 0` now.
> 
> But yea not a big deal.

Thanks, yeah no I completely missed the printed part. I fixed it though now :laughing:

**Review by thozza — APPROVED** [review]

**Pull request merged**